lib/hermes/message_queue_listener.rb

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env ruby
+require './config/environment.rb'
+require 'deepstruct'
+
+module Hermes
+
+  # Listens for a new post.hermes_message, looks up provider,
+  # dispatches, and updates the grove post with a tag reflecting new status
+  class MessageQueueListener
+
+    def call(message)
+      consider message.payload
+      nil
+    end
+
+
+    def consider(payload)
```

Comment:
Not sure... renamed the method and moved the event type check in order justify its continued existence :)